server/views/partials/snooze-banner.ejs

```diff
@@ -1,15 +1,15 @@
 <div class="moj-banner govuk-!-margin-left-2 govuk-!-margin-right-2 govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-three-quarters">
 
     <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
       <path d="M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z" />
     </svg>
 
     <div class="moj-banner__message">
-      <h2 class="govuk-heading-s govuk-!-padding-bottom-2">Notifications paused until <%= snoozeUntil %></h2>
+      <h2 class="govuk-heading-s govuk-!-padding-bottom-2">Your notifications are currently turned off. You have set them to start again on <%= snoozeUntil %>. Or, you can choose to resume them at any time.</h2>
```

Comment:
Well, this is going to break my integration tests 😂

Reply:
Sorry. Our _copy_ chap did a thorough job on it though 👍